keyboards/just60/readme.md

```diff
@@ -0,0 +1,17 @@
+# Just60
+
+Just60 keyboard produced by Yang. The keyboard comes with a custom Mass Storage Device bootloader and a TMK based firmware from ydkb.io.
+
+To use a QMK based firmware, you might want install a QMK bootloader. The PCB exposes 6 pins for ISP(In-System Programming), and they are located just under the ATMega32U4 chip. From left to right, the pins are `VCC`, `SCLK`, `MOSI`, `MISO`, `RESET`, `GND`. The `GND` is always the square one. You could program the flash with any AVR programmer, or a Raspberry Pi with `avrdude`.
```

Comment:
```suggestion
To use a QMK based firmware, you might want install a QMK bootloader. The PCB exposes 6 pins for ISP(In-System Programming), and they are located just under the ATMega32U4 chip. From left to right, the pins are `VCC`, `SCLK`, `MOSI`, `MISO`, `RESET`, `GND`. The `GND` is the square one. You could program the flash with any AVR programmer, or a Raspberry Pi with `avrdude`.
```

A square pad typically only denotes pin 1, whose purpose depends on the application. For example, most AVR ISP headers use this standard pinout instead, where MISO is pin 1:
![](https://lh4.googleusercontent.com/_N4orY3Ztc3M/Tcq0gtaX_gI/AAAAAAAAAKM/9fju_il9l5U/s800/avrispmkII-pin-out.jpg)